src/bcs/SakiyamaElectronDiffusionBC.C

```diff
@@ -0,0 +1,95 @@
+//* This file is part of Zapdos, an open-source
+//* application for the simulation of plasmas
+//* https://github.com/shannon-lab/zapdos
+//*
+//* Zapdos is powered by the MOOSE Framework
+//* https://www.mooseframework.org
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "SakiyamaElectronDiffusionBC.h"
+
+registerMooseObject("ZapdosApp", SakiyamaElectronDiffusionBC);
+
+template <>
+InputParameters
+validParams<SakiyamaElectronDiffusionBC>()
+{
+  InputParameters params = validParams<IntegratedBC>();
+  //params.addRequiredParam<Real>("r", "The reflection coefficient");
+  params.addRequiredCoupledVar("mean_en", "The mean energy.");
+  params.addRequiredParam<Real>("position_units", "Units of position.");
+  params.addClassDescription(
+    "Kinetic electron boundary condition"
+    "(Based on DOI: https://doi.org/10.1116/1.579300)");
+  return params;
+}
+
+SakiyamaElectronDiffusionBC::SakiyamaElectronDiffusionBC(const InputParameters & parameters)
+  : IntegratedBC(parameters),
+
+    _r_units(1. / getParam<Real>("position_units")),
+    //_r(getParam<Real>("r")),
```

Comment:
Remove comment (and addRequiredParam comment above) if not needed any more.

Reply:
Removed.